job-descriptions/software-engineer-intern.md

```diff
@@ -11,26 +11,22 @@ We have already built the world's best [code search](https://about.sourcegraph.c
 
 Read [our blog](https://about.sourcegraph.com/blog/) to see what more we have been working on recently and read [our master plan](https://sourcegraph.com/plan) to see where we are going!
 
-### About the team
-
-Our small team consists of talented, mature, collaborative, driven individuals who are attracted to the massive problem we are tackling. We work in an open environment that treats people in a first-class manner and provides them with ownership, responsibility, and autonomy.
-
 ### About the role
 
-*This is a paid internship* 
+_This is a paid internship_
 
 As an intern, you will help build Sourcegraph, a multi-tier application (web, CLI, browser extensions, API, data stores, services) written primarily in TypeScript (frontend) and Go (backend). You will be matched with a mentor who will help plan your work, and then you will have responsibility and autonomy to get the work done.
 
-Ideal candidates:
-
-- Are pursuing a computer science undergraduate or graduate degree.
-- Can demonstrate passion and excitement about developer tools (e.g. have built a developer tool, contributed to open source developer tools or other open source projects, written blog posts or talks about development, etc.)
-- Have submitted a PR in any one of our repos (and linked to it in their application)!
-
 Example intern projects from past years:
 
 - Build a browser extension from scratch that communicates with the Sourcegraph API to inject code intelligence on http://github.com.
 - Build a Sourcegraph electron application that is synced with the cursor position in your editor via an editor extension.
 - Integrate Sourcegraph search into Alfred.
 
+### Ideal candidates:
+
+- Are pursuing a computer science undergraduate or graduate degree.
+- Can demonstrate passion and excitement about developer tools (e.g. have built a developer tool, contributed to open source developer tools or other open source projects, written blog posts or talks about development, etc.)
+- _(Not required)_ Have submitted a PR in any one of our repos (and linked to it in their application)!
```

Comment:
Let's just delete this line here. The actual application gives candidates an opportunity to link to contributions.